File: java/src/test/java/codingdojo/FizzBuzzTest.java
package codingdojo;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class FizzBuzzTest {
    @Test
    void shouldReturnHelloWorld() {
        // given

        // when
        String actualResult = Fizzbuzz.fizzbuzz();

        // then
        assertEquals("hello world", actualResult);
    }
}

File: kotlin/src/test/kotlin/codingdojo/FizzBuzzTest.kt
package codingdojo;

import org.junit.Test
import kotlin.test.assertEquals

class FizzBuzzTest {
    @Test
    fun `should return hello world`() {
        // given

        // when
        val actualResult = fizzbuzz()

        // then
        assertEquals("hello world", actualResult)
    }
}
